lib/skin.js

```diff
@@ -27,29 +35,74 @@ module.exports = function () {
     }
   }
 
-  function createSkin(skinId, baseResponsiveSkins) {
-    var skinDir = path.resolve('res/skins', skinId);
-    return new Skin(skinId, skinDir, baseResponsiveSkins);
+  function createSkin(skinId, skinDir) {
+    return new Skin(skinId, skinDir);
   }
 
-  var coreResponsiveSkinIds = ['responsive_peak'];
+  var coreResponsiveSkinIds = ['responsive_peak', 'responsive_base'];
+  //Subset of legacy core skins -- This list is fetched from studio's subset of skins exposed from skin editor.
+  var coreSkinIds = ["base", "whiteui", "proui", "pacificaui", "newsui", "naturalbridgesui", "mondrian", "lightui",
+                    "castlerockui", "anonuevoui" ];
 
```

Comment:
How do we remember to update this when a new core skin gets added? I guess this will be just manual right now, but we should document it or keep a note somewhere to remind us in the future.